python/pyarrow/tests/test_compute.py

```diff
@@ -109,6 +109,77 @@ def test_sum_chunked_array(arrow_type):
     assert pc.sum(arr).as_py() is None  # noqa: E711
 
 
+@pytest.mark.parametrize("arrow_type", numerical_arrow_types)
```

Comment:
We don't need to test multiple types and values agggressively, since that it already done on the C++ side. Mostly we should check that the Python wrapper has the expected API and return type.